src/main/java/fr/polytech/sim/log/LoggerDecorator.java

```diff
@@ -0,0 +1,7 @@
+package fr.polytech.sim.log;
+
+public abstract class LoggerDecorator implements Logger {
+
+    protected Logger logger;
```

Comment:
Ce champ peut être final, mais ca demande des modifications au code, en ajoutant un constructeur.

```suggestion
    protected final Logger logger;

    protected LoggerDecorator(Logger logger) {
        this.logger = logger;
    }
```